src/Annotator/ClassAnnotatorTask/TableFindAnnotatorTask.php

```
@@ -0,0 +1,112 @@
+					}
+				}
+			}
+		});
+
+		$modifiedAst = $traverser->traverse($ast);
+		$prettyPrinter = new PrettyPrinter();
+		$modifiedCode = $prettyPrinter->prettyPrintFile($modifiedAst);
+
+		dd($modifiedCode);
+
+		return $this->annotateInlineContent($path, $this->content, $annotations, $rowToAnnotate);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$annotations'. (Unused Code Rules)

(UnusedLocalVariable)